app/assets/stylesheets/pre_registrations.css.scss

```diff
@@ -1,3 +1,24 @@
-// Place all the styles related to the Pre_registrations controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+table {
+    font-family: arial, sans-serif;
+    border-collapse: collapse;
+    width: 100%;
+}
+
+th {
```

Comment:
It will be better to use a custom class with _th_ or _td_ tags and add the corresponding CSS against those classes.